v10/index.html

```diff
@@ -86,7 +86,15 @@
 	<svg id="docs-icon-window-application-6" width="24" height="24" viewBox="0 0 24 24"><path d="M9 8.002H2V19c0 1.103.897 2 2 2h5V8.002zM7 19.003H4v-2h3v2zm0-3H4v-2h3v2zm-3-3v-2h3v2H4zm18-6.001V5c0-1.103-.896-2-2-2H4c-1.103 0-2 .897-2 2v2.002h20zM8 4a1 1 0 1 1 0 2 1 1 0 1 1 0-2zM5 4a1 1 0 1 1 0 2 1 1 0 1 1 0-2zm6 4.002V21h9c1.104 0 2-.897 2-2V8.002H11zm4 10.001h-2v-3h2v3zm5 1h-4v-4h4v4zm0-6h-7v-2h7v2z" fill="#757575"/></svg>
 	<svg id="docs-icon-focus-2" width="24" height="24" viewBox="0 0 24 24"><path d="M3 5h3V3H3c-1.104 0-2 .897-2 2v3h2V5zm18-2h-3v2h3v3h2V5c0-1.103-.896-2-2-2zM3 16H1v3a2 2 0 0 0 2 2h3v-2H3v-3zm18 3h-3v2h3a2 2 0 0 0 2-2v-3h-2v3zM5 7h14v10H5z" fill="#757575"/></svg>
 </svg>
-
+		<style id="antiClickjack">body{display:none !important;}</style>
```

Comment:
Could you move this up to be the very first thing that loads? Just to make sure nothing can mess with it